engines/windowsaio.c

```diff
@@ -182,8 +182,7 @@ static int fio_windowsaio_open_file(struct thread_data *td, struct fio_file *f)
 	 * Inform Windows whether we're going to be doing sequential or
 	 * random io so it can tune the Cache Manager
 	 */
-	if (td->o.td_ddir == TD_DDIR_READ  ||
-		td->o.td_ddir == TD_DDIR_WRITE)
+	if (!td_random(td))
```

Comment:
(Not introduced by this commit but an idle comment) Technically shouldn't this only be done if the user has set [`fadvise_hint`](http://fio.readthedocs.io/en/latest/fio_doc.html#cmdoption-arg-fadvise-hint) ?